templates/partials/_country-field.html

```diff
@@ -0,0 +1,259 @@
+<label for="country">Country:</label>
```

Comment:
```suggestion
<label class="is-required" for="country">Country:</label>
```
We can add `is-required` class to show (*) on the label